docs/internals/dev.rst

```diff
@@ -58,22 +57,15 @@ Python "venv" with all dependencies and commands installed into it.
       $ python3.9 -m venv edgedb-dev
       $ source edgedb-dev/bin/activate
 
-#. Build edgedb-python:
-
-   .. code-block:: bash
-
-      $ cd edgedb-python
-      $ pip install -v -e .
-
 #. Build edgedb (the build will take a while):
 
    .. code-block:: bash
 
       $ cd ../edgedb
-      $ pip install -v -e ".[test,docs]"
+      $ pip install -v -e ".[test]"
```

Comment:
Why removing `docs`?

Reply:
Because `test` implies `docs` in `setup.py`.